collector/setting.go

```diff
@@ -0,0 +1,31 @@
+package collector
+
+import (
+	rancher "github.com/rancher/types/client/management/v3"
+	log "github.com/sirupsen/logrus"
+)
+
+func SetSetting(client *rancher.Client, key string, value string) error {
+	setting, err := client.Setting.ByID(key)
+	if err == nil {
```

Comment:
Should ensure that error is an exists-error. Might be unrelated error when setting doesn't exist yet.

Reply:
Fixed.